app/istqb-ct-mbt-complete-guide/istqb-ct-mbt-complete-guide.css

```
@@ -0,0 +1,976 @@
+    @media (max-width: 600px) {
+        .hero-meta   { gap: 1rem; }
+        .metric-grid { grid-template-columns: 1fr 1fr; }
+        .exam-grid   { grid-template-columns: 1fr 1fr; }
+        .ref-grid    { grid-template-columns: 1fr; }
+    }
+
+    /* ===== Scrollbar ===== */
+    ::-webkit-scrollbar       { width: 6px; height: 6px; }
+    ::-webkit-scrollbar-track { background: var(--color-bg-primary, #0a0e1a); }
+    ::-webkit-scrollbar-thumb { background: rgba(79, 209, 197, 0.3); border-radius: 3px; }
+    scrollbar-width: thin;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)